src/main/java/pl/h2security/registration/RegistrationController.java

```diff
@@ -0,0 +1,24 @@
+package pl.h2security.registration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import pl.h2security.user.User;
+
+@RestController
+public class RegistrationController {
+
+private RegistrationService registrationService;
+
+    @Autowired
+public RegistrationController(RegistrationService registrationService){
+        this.registrationService=registrationService;
+    }
+
+    @PostMapping("/register")
+    public void registerUser(@RequestBody User user){
```

Comment:
We could return some response to the client. We can use ResponseEntity wrapper class for it just to lett the client know that registration went well. Let's return http status ok when everything goes well.

Reply:
https://www.baeldung.com/spring-response-entity